samples/ingestion/ingestion-client/FetchTranscription/Language/AnalyzeConversationsProvider.cs

```diff
@@ -105,30 +139,47 @@ public async Task<(IEnumerable<string> jobIds, IEnumerable<string> errors)> Subm
                                     }
                                 }
                             }
-                        }
+                        },
                     });
                 }
 
-                data.Last().AnalysisInput.Conversations[0].ConversationItems.Add(new ConversationItem
+                var utterance = new ConversationItem
                 {
                     Text = topResult.Display,
                     Lexical = topResult.Lexical,
                     Itn = topResult.ITN,
                     MaskedItn = topResult.MaskedITN,
                     Id = $"{turnCount}__{recognizedPhrase.Offset}__{recognizedPhrase.Channel}",
                     ParticipantId = $"{recognizedPhrase.Channel}",
+                    Role = (turnCount % 2 == 0) ? "Agent" : "Customer",
+                    ConversationItemLevelTiming = new AudioTiming
+                    {
+                        Offset = recognizedPhrase.OffsetInTicks,
+                        Duration = recognizedPhrase.DurationInTicks,
+                    },
                     AudioTimings = topResult.Words
                         ?.Select(word => new WordLevelAudioTiming
                         {
                             Word = word.Word,
                             Duration = (long)word.DurationInTicks,
                             Offset = (long)word.OffsetInTicks
                         })
-                });
+                };
+                data.Last().AnalysisInput.Conversations[0].ConversationItems.Add(utterance);
+                summarizationData.AnalysisInput.Conversations[0].ConversationItems.Add(utterance);
                 count += textCount;
                 turnCount++;
             }
 
+            this.log.LogInformation("{0} Summarization Tasks Prepared. Locale = {1}. chars = {2}. turns = {3}.", summarizationData.Tasks.Count, this.locale, count, turnCount);
```

Comment:
nit: we use string interpolation instead of composite formatting everywhere else in the code. consider changing the logs to be consistent